### PR TITLE
CCM-5850: Review code owners

### PR DESCRIPTION
**Pull request by m-salaudeen**
## Type of changes

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Comment by mark-r-bjss** [comment]
We should be following the pattern that has been established in the template management repo: https://github.com/NHSDigital/nhs-notify-web-template-management/blob/main/.github/CODEOWNERS 

The equivalent teams are:
https://github.com/orgs/NHSDigital/teams/nhs-notify-iam-webauth
https://github.com/orgs/NHSDigital/teams/nhs-notify-iam-webauth-testers

File: project.code-workspace [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "folders": [
     {
-      "name": "NHS Notify Repo Template",
+      "name": "NHS Notify Web IAM Template",
```
Comment (mark-r-bjss) [inline project.code-workspace:4]: Delete the word "Template"

Comment (m-salaudeen) [reply project.code-workspace]: Done

**Review by mark-r-bjss — APPROVED** [review]

**Review by alibombaye — APPROVED** [review]

**Review by harrim91 — APPROVED** [review]

**Review by mark-r-bjss — APPROVED** [review]

File: .github/workflows/stage-2-test.yaml [hunk 2]
```diff
@@ -86,7 +86,7 @@ jobs:
     permissions:
       id-token: write
       contents: read
-    timeout-minutes: 5
+    timeout-minutes: 10
```
[the remark this answers is not in the document]
Comment (m-salaudeen) [reply .github/workflows/stage-2-test.yaml]: Done

**Review by mark-r-bjss — APPROVED** [review]

**Pull request merged**